public/locales/en.json

```diff
@@ -387,9 +387,8 @@
       "description": "A printed copy of the ENS constitution and its signers is now available in hardcover and ultra-limited edition of 50"
     },
     "swag": {
-      "title": "ENS swag at devcon Bogota",
+      "title": "ENS swag at Devcon Bogota",
```

Comment:
I changed this since Devcon as a brand should be capitalized, however, I couldn't find any brand guidelines from them :/